generators/generator-base.js

```diff
@@ -472,6 +472,7 @@ module.exports = class extends PrivateBase {
             { name: 'Greek', value: 'el' },
             { name: 'Hindi', value: 'hi' },
             { name: 'Hungarian', value: 'hu' },
+            { name: 'Bahasa Indonesia', value: 'id' },
```

Comment:
Sometime people say 'Bahasa', sometime 'Indonesian' and sometime 'Bahasa Indonesia'. 
'Bahasa' means 'Language' in english.